common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1536,6 +1537,7 @@ public static final class BrokerContentProvider {
         public enum API {
             MSAL_HELLO(MSAL_HELLO_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_INTERACTIVE(MSAL_ACQUIRE_TOKEN_INTERACTIVE_PATH, null, VERSION_3),
+            ACQUIRE_TOKEN_ACCOUNT_TRANSFER_INTERACTIVE(MSAL_ACQUIRE_TOKEN_ACCOUNT_TRANSFER_INTERACTIVE_PATH, null, VERSION_8),
```

Comment:
protocol version 14.0?

https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=/%5BAndroid%5D%20Broker%20API/broker_protocol_versions.md&_a=preview

(not sure if this docs is updated btw)

Reply:
No longer needed